x-pack/plugins/logstash/common/constants/monitoring.js

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const MONITORING = {
+
+  /**
+   * How far back in time (from now) do we look in Monitoring data for pipelines to be considered as "currently" running. This is
+   * also, deliberately, the same duration we give Monitoring to pick up and report on a recently-deleted pipeline before we
+   * are safe to stop tracking that pipeline as recently-deleted.
+   */
+  PIPELINE_RECENCY_DURATION_S: 30
```

Comment:
Thanks for adding the comment, it's very helpful!

I think `ACTIVE_PIPELINE_RANGE_S` is a slightly better name. It denotes that this constant represents a timespan for active pipeline detection. A more descriptive name could be `ACTIVE_PIPELINE_POLL_RANGE_S`. WDYT @ycombinator

Reply:
I like `ACTIVE_PIPELINE_RANGE` as it works for both usages of this constant. Thanks, will change!